tests/metrics/test_data.py

```diff
@@ -712,3 +713,65 @@ def test_get_github_build_metrics():
     assert isinstance(result["total_runs"], int)
     assert isinstance(result["successful_runs"], int)
     assert isinstance(result["failing_runs"], int)
+
+
+@pytest.mark.parametrize(
+    "content, expected",
+    [
+        # Test case: Single platform (Twitter)
+        (
+            "Follow us on Twitter: https://twitter.com/ourproject",
+            {"social_media_platforms": ["Twitter"], "social_media_platforms_count": 1},
+        ),
+        # Test case: Multiple platforms
+        (
+            """Connect with us:
+            - Twitter: https://twitter.com/ourproject
+            - LinkedIn: https://linkedin.com/company/ourcompany
+            - Discord: https://discord.gg/invitecode
+            """,
+            {
+                "social_media_platforms": ["Discord", "LinkedIn", "Twitter"],
+                "social_media_platforms_count": 3,
+            },
+        ),
+        # Test case: No social media links
+        (
+            "This README.md contains no social media links.",
+            {"social_media_platforms": [], "social_media_platforms_count": 0},
+        ),
```

Comment:
I believe your regexes exclude links to platforms that don't include a specific user, channel, etc. so that might be a nice thing to test here as well. For example, you could check that just referencing `https://www.youtube.com` doesn't count as a hit.

Reply:
Thanks great point! I'll add in another test case for this.